src/main/java/io/swagger/codegen/v3/generators/java/AbstractJavaCodegen.java

```diff
@@ -50,6 +50,7 @@
     public static final String JAVA8_MODE = "java8";
     public static final String WITH_XML = "withXml";
     public static final String SUPPORT_JAVA6 = "supportJava6";
+    private static final String NOT_NULL_JACKSON_ANNOTATION = "notNullJacksonAnnotation" ;
```

Comment:
the flag needs to be added also as cliOption

Reply:
done